backend/kesaseteli/applications/models.py

```diff
@@ -125,7 +125,7 @@ def send_activation_email(self, request):
             message=self._activation_email_message(request),
             from_email=settings.DEFAULT_FROM_EMAIL,
             recipient_list=[self.email],
-            fail_silently=False,
+            fail_silently=True,
```

Comment:
I think it would make sense to check the return value when fail_silently is used, and log a message/warning if it is 0.
"The return value will be the number of successfully delivered messages (which can be 0 or 1 since it can only send one message)."

Reply:
Taken into account by logging an error if the return value from send_mail is zero.